src/main/java/Duke.java

```diff
@@ -1,10 +1,169 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    /**
+     * Function checks if the user's 2nd word input is a numeral value whenever the 1st word input is "mark" or "unmark".
+     *
+     * @param str the 2nd word input by the user whenever the 1st word input is "mark" or "unmark"
+     * @return boolean true if the string is a numeral, and false if it contains any other characters.
+     */
+    public static boolean checkIsNumeric(String str) {
+        for (int i = 0; i < str.length(); i++) {
+            char c = str.charAt(i);
+            if (c < '0' || c > '9') {
+                return false;
+            }
+        }
+        return true;
+    }
+
+    /**
+     * Returns the number of words within a string.
+     *
+     * @param input the string input from the user
+     * @return the number of words in the string
+     */
+
+    public static int getNumOfWords(String input) {
+        if (input == null || input.isEmpty()) {
+            return 0;
+        }
+        String[] words = input.split("\\s+");
+        return words.length;
+    }
+
+    public static void printNumOfTasks(int numOfTasks) {
```

Comment:
I like how you name the parameters. It was clear when the number of tasks is different which of the corresponding line is printed.